Automattic/jetpack
# Protect: Integrate Brute Force Protection

This is the primary project branch for integrating Brute Force Protection into the Jetpack Protect plugin.

## Proposed changes:

See the project milestone for a list of all PRs that have been reviewed and merged into this project branch: https://github.com/Automattic/jetpack/milestone/304?closed=1

### Other information:


## Jetpack product discussion



peb6dq-oP-p2

## Does this pull request change what data or activity we track or use?



Yes. New tracks events are added - see https://github.com/Automattic/jetpack/pull/29548.

## Testing instructions:






* Review each merged PR from [the milestone](https://github.com/Automattic/jetpack/milestone/304?closed=1) and verify it has been approved by the appropriate team.
* Test the Brute Force Protection feature via the Protect plugin. See peb6dq-QV-p2 for detailed testing instructions.
